netlogo-gui/src/test/app/infotab/InfoFormatterTests.scala

```diff
@@ -104,13 +105,13 @@ class InfoFormatterTests extends FunSuite {
 
   // in contrast, here no <pre> because no blank line
   test("fenced code blocks 2") {
-    assertResult("""|<p>foo<br/><code>
-              |   bar
-              |  baz
-              |qux
-              |</code><br/>oof</p>""".stripMargin) {
+    assertResult("""|<p>foo<br/><code class="text">
```

Comment:
What's the purpose of this test, then? Does it check inline code? If so, the test itself would better be changed to be more clear?

I'll also remove the comment since it's misleading.